### PR TITLE
add reset component

### PR DESCRIPTION
**Pull request by nguasch**
Please, review @larnedo @PablitoGS @nucliweb

### CODE REVIEW

**Comment by PablitoGS** [comment]
:+1:

**Comment by nucliweb** [comment]
We need the following _devDependencies_ `node-sass`, `postcss-cli` & `autoprefixer`.

@nguasch please add the dependencies in `package.json`, thanks

**Comment by nguasch** [comment]
done @nucliweb

File: packages/reset/src/_reset.scss [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
```
Comment (nucliweb) [inline packages/reset/src/_reset.scss]: Please change the double quotes for single quotes, thanks.
 `'` ➔ `"`

**Comment by nguasch** [comment]
done, @nucliweb

**Comment by nucliweb** [comment]
:+1:

**Pull request merged**